translation_en/chapter_08_EN.md

```diff
@@ -67,15 +67,15 @@ As of writing this book, there is no clear consensus or definition about the ter
 
   
 
-> "Cloud-native technologies enable companies to create and run scalable applications in modern and dynamic environments, such as public, private, and hybrid clouds. Containers, service meshes, microservices, immutable infrastructure, and declarative APIs exemplify this approach. (...) "
+> "Cloud-native technologies enable companies to create and run scalable applications in modern and dynamic environments, such as public, private, and hybrid clouds. Containers, service meshes, microservices, immutable infrastructure, and declarative APIs exemplify this approach. (...)"
 
 >
 
 > — [Cloud-Native Computing Foundation]([https://www.cncf.io/](https://www.cncf.io/))
 
   
 
-> "Cloud-native é um adjetivo que descreve as aplicações, arquiteturas, plataformas/infraestrutura e processos, que juntos as fazem funcionar de uma forma que nos permita melhorar nossa habilidade de rapidamente responder a mudanças e reduzir imprev that together make it economical to work in a way that allows us to improve our ability to quickly respond to change and reduce imprevisibilidade.” POR FAVOR, VERIFIQUEM ESSE PARÁGRAFO, MISTURA PORT. E ING.
+> "'Cloud native' is an adjective that describes the applications, architectures, platforms/infrastructure, and processes, that together make it *economical* to work in a way that allows us to improve our ability to quickly respond to change and reduce unpredictability"
```

Comment:
Essa frase deveria ficar entre aspas? A frase não é o texto exato que consta no arquivo... é uma reescrita to que foi dito por Posta neste link...